### PR TITLE
Update hint localize for translation + new strings in stringtabel.xml for translations

### PR DESCRIPTION
**Pull request by WolfSkin1337**
<Please review the guidelines for contributing to this repository. The link is above.>

### **at `life_server/functions.sqf`**
- Update `hint` to `hint localize`

### **at `Altis_Life.Altis/stringtable.xml`**
- Add Strings in `Stringtable.xml` for `hint changes` in  `life_server\functions.sqf`:
- - Add new Package
- - `Package name="CellPhone_Messages"`
- Add the newly defined strings for `life_server\functions.sqf`
- Add German translation for the newly defined strings

### CODE REVIEW

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -1278,6 +1278,85 @@
             <Portuguese>Chamar SAMU</Portuguese>
             <Polish>MEDYK</Polish>
         </Key>
+	</Package>
```
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: This line is indented using a tab. It should be indented via 4 spaces per tab as per [contributing](https://github.com/AsYetUntitled/Framework/blob/master/.github/CONTRIBUTING.md#help-the-travis-ci-build-failed). 

For an unknown reason [Travis CI did not catch this](https://travis-ci.org/AsYetUntitled/Framework/builds/189383459) as it passed. Ping, @BoGuu.

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -1278,6 +1278,85 @@
...
+	<Package name="CellPhone_Messages">
```
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Ditto.

**Comment by WolfSkin1337** [comment]
Fixed the tabs and add the Placeolders.
so when its down i go to  #120 and help there.

**Comment by BoGuu** [comment]
LGTM

**Comment by WolfSkin1337** [comment]
Update Placeholders

**Comment by Jawshy** [comment]
LGTM - partially tested with no issues found as of https://github.com/AsYetUntitled/Framework/pull/223/commits/2a75ee54c93f468719d5d806080c7242f5b6f221.

**Pull request merged**